### ISSUE TITLE
Unable to give user admin rights to view

### ISSUE THREAD

**Issue by wknapik** [humio/issues#78]
* Humio Version
https://cloud.humio.com/ (v1.5.4)
* Browser Version
  * curl 7.64.0 (x86_64-pc-linux-gnu) libcurl/7.64.0 OpenSSL/1.1.1b zlib/1.2.11 libidn2/2.1.1 libpsl/0.20.2 (+libidn2/2.1.1) libssh2/1.8.1 nghttp2/1.36.0
  * Chromium 73.0.3683.86 Arch Linux

```
mutation {
  addMember(searchDomainName: "foo", username: "bar", hasDeletionRights: false, hasMembershipAdminRights: true) {
    clientMutationId
  }
}

```

produces

```
{
  "data": null,
  "errors": [
    {
      "message": "You are not allowed to change deletion rights on this repository.",
      "path": [
        "addMember"
      ],
      "locations": [
        {
          "column": 3,
          "line": 2
        }
      ]
    }
  ]
}
```

when running with full admin rights to the view.

The following combinations work:
* `hasDeletionRights: false, hasMembershipAdminRights: false`
* `hasDeletionRights: true, hasMembershipAdminRights: false`

The following combinations fail:
* `hasDeletionRights: true, hasMembershipAdminRights: true`
* `hasDeletionRights: false, hasMembershipAdminRights: true`

Notice that the error message refers to deletion rights, even when only admin rights are being enabled.

Either I'm missing something, or this is a bug.

**Comment by pmech** [comment]
Hi @wknapik . Thank you for the bug report. I'll investigate and get back to you.

**Comment by pmech** [comment]
As you indicated :) this is a bug in our end. It will be fixed in the next release (some time this week I expect).

Thank you very much for reporting this! and let us know if you hit other issues or just have questions.

Regards

**Comment by wknapik** [comment]
Thanks!

**Comment by chvitved** [comment]
The fix will go out with next release 1.5.5
